test/extensions/filters/http/admission_control/default_evaluator_test.cc

```diff
@@ -0,0 +1,180 @@
+#include <chrono>
+
+#include "envoy/extensions/filters/http/admission_control/v3alpha/admission_control.pb.h"
+#include "envoy/extensions/filters/http/admission_control/v3alpha/admission_control.pb.validate.h"
+
+#include "extensions/filters/http/admission_control/admission_control.h"
+#include "extensions/filters/http/admission_control/evaluators/default_evaluator.h"
+
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::NiceMock;
+using testing::Return;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AdmissionControl {
+namespace {
+
+class DefaultEvaluatorTest : public testing::Test {
+public:
+  DefaultEvaluatorTest() = default;
+
+  void makeEvaluator(const std::string& yaml) {
+    AdmissionControlProto::DefaultEvaluationCriteria proto;
+    TestUtility::loadFromYamlAndValidate(yaml, proto);
+
+    evaluator_ = std::make_unique<DefaultResponseEvaluator>(proto);
+  }
+
+  void expectHttpSuccess(int code) { EXPECT_TRUE(evaluator_->isHttpSuccess(code)); }
+
+  void expectHttpFail(int code) { EXPECT_FALSE(evaluator_->isHttpSuccess(code)); }
+
+  void expectGrpcSuccess(int code) { EXPECT_TRUE(evaluator_->isGrpcSuccess(code)); }
+
+  void expectGrpcFail(int code) { EXPECT_FALSE(evaluator_->isGrpcSuccess(code)); }
+
+  void verifyGrpcDefaultEval() {
+    // Ok.
```

Comment:
i would make the helper take the enum value so you dont need to add a comment saying which one it is